README.md

```diff
@@ -129,7 +129,7 @@ Typst's CLI is available from different sources:
     `nix run github:typst/typst -- --version`.
 
 - Docker users can run a prebuilt image with
-  `docker run -it ghcr.io/typst/typst:latest`.
+  `docker run ghcr.io/typst/typst:latest --help`.
```

Comment:
Do we really need the `:latest` part? I think it should be dropped.

Reply:
It's pretty standard in the Docker scene. I see no reason to drop it.

Reply:
The reason is the shorter command, as it makes the command unnecessary verbose.

I like it clean and short. Ahem...